models/patchArticle.js

```
@@ -1,16 +1,17 @@
 const db = require('../db/connection.js');
 
 const patchArticle = (votes, articleId) => {
-    return db.query(`UPDATE articles SET votes = (votes + $1) WHERE article_id = $2 RETURNING *;`, [votes, articleId])
+   
+ 
+  if(!Number.isInteger(votes)){
+    return Promise.reject({status: 400, msg: 'Invalid input'});
+  }
+  
```

In theory this is fine, but psql should return an error for us that we can handle anyway. If you remove the catch block at the bottom of this function on line 16, you should catch a psql error in your controller. See if you console.log err.code inside your catch block in your controller whether you get anything back!